contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsFetcher.java

```diff
@@ -348,6 +360,9 @@ private File fetchFromSource(String sourceFileUrl,
             if(stats != null) {
                 stats.complete();
             }
+            if (!isCompleteFetch) {
```

Comment:
I think most of our code has a space between the operator and the operand. For consistency, should probably put a space between ! and isCompleteFetch.